src/vishakha/WhileLoopExample.java

```diff
@@ -0,0 +1,55 @@
+package vishakha;
+
+public class WhileLoopExample {
+	
+	void printEvenNos(int startIndex, int endIndex){
+		System.out.println("Even Numbers are : ");
+		int index=startIndex;
```

Comment:
Here, you dont need to write extra variable, you can use startIndex variable for increment.